frontend/javascripts/viewer/view/ai_jobs/alignment/ai_image_alignment_job.tsx

```diff
@@ -9,6 +11,20 @@ export const AiImageAlignmentJob = () => {
     <AlignmentJobContextProvider>
       <Flex gap={24}>
         <Flex flex="2" vertical gap={24}>
+          <Card
+            type="inner"
+            title={
+              <Space align="center">
+                <InfoCircleOutlined style={{ color: ColorWKBlue }} />
+                Align Sections
+              </Space>
+            }
+          >
+            <Typography.Text>
+              Align sections of your dataset to correct for shifts and rotations. Select an
```

Comment:
strictly speaking, scalings, shearings etc are also supported. maybe "to correct for irregularities (e.g., shifts and rotations)"?